src/test/java/BaseTest.java

```diff
@@ -1,18 +1,137 @@
+import com.beust.jcommander.Parameter;
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
 
+import java.time.Duration;
 
-public class BaseTest {
 
+public class BaseTest {
+    public static WebDriver driver = null;
+    WebDriverWait wait;
+    By submitBtn = By.cssSelector("[type='submit']");
+    By nextBtn = By.cssSelector("[data-testid ='play-next-btn']");
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
 
-}
+
+    @BeforeMethod
+    @Parameters("BaseUrl")
+    public void setUpBrowser(String BaseUrl) {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        driver.get(BaseUrl);
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+    }
+
+   protected void openBrowser() {String url = "https://bbb.testpro.io/";
+        driver.get(url);
+   }
+
+    public void closeBrowser() {
+        driver.quit();
+    }
```

Comment:
The @AfterMethod annotated method will be invoked after the execution of each test method.

```
  @AfterMethod
  public void closeBrowser() {
    driver.quit();
  }
```